src/content-helper/editor-sidebar/class-editor-sidebar.php

```diff
@@ -8,18 +8,22 @@
 
 declare(strict_types=1);
 
-namespace Parsely;
+namespace Parsely\Content_Helper;
 
+use Parsely\Parsely;
 use Parsely\Content_Helper\Content_Helper_Feature;
 
 use function Parsely\Utils\get_asset_info;
 
+use const Parsely\PARSELY_FILE;
+
 /**
  * Class that generates and manages the PCH Editor Sidebar.
  *
  * @since 3.5.0
+ * @since 3.9.0 Renamed FQCN from `Parsely\Content_Helper` to `Parsely\Content_Helper\Editor_Sidebar`
```

Comment:
What does it mean by `FQCN`?

Reply:
FQCN = Fully Qualified Class Name, which can be translated to "the full class name, including its full namespace path". In this specific instance, we haven't just changed the class name but also changed the namespace it belongs to, thus the reference to FQCN instead of saying "renamed class".

Reply:
Thanks for explaining but still I likes simple `class name` better here ..... feel free to keep this change if you think its better

Reply:
FQCN is standard terminology in multiple languages. The use is correct here.